### DISCUSSION TITLE
New Shit, Y'all

### DISCUSSION THREAD

**Discussion by DannybotDev** [FunnyDannyG/FNFMegaMayhem#13] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/DannybotDev/FNFMegaMayhem/releases/tag/v2.0.0prealpha'>New Shit, Y'all</a>.</em>

**Comment by JordiWifBread** [comment]
flarp in sharty waett

**Reply by DannybotDev** [reply]
eat a shit
